strutil/strutil_test.go

```diff
@@ -46,6 +46,13 @@ func TestStrListDelete(t *testing.T) {
 	if StrListContains(output, "item3") {
 		t.Fatal("bad: 'item3' should not have been present")
 	}
+
+	// Test that StrListDelete doesn't modify its input
+	input := []string{"a", "b", "c"}
+	output = StrListDelete(input, "b")
```

Comment:
Nit: the `output` variable is unused below, should it be replaced with `_` or is compiler OK with it?